README.md

```diff
@@ -1,257 +1,116 @@
 
-### Validating a Petition
+### User types
 
-<a href="docs/images/screenshot_petition.png"><img src="docs/images/screenshot_petition.png" width="80%" style="float: left; margin: 4px 4px 0 0;" /></a>
+- **Petitioners** - Citizens filing a petition with the court; if filing without legal representation, referred to as pro-se petitioner
+- **Practitioners** - Attorneys who file a petition on behalf of a petitioner
+- **Internal court users** - System users responsible for managing cases within the system, work at Tax Court
+- **IRS attorneys** - Work for IRS and file answers to petitions
+- **General public** - Can search for a case on public website and see certain details about it that are open to the public
 
-### Trial session details
+### Project status
 
-<a href="docs/images/screenshot_trialsession.png"><img src="docs/images/screenshot_trialsession.png" width="80%" style="float: left; margin: 4px 0 0 4px;" /></a>
+DAWSON is in active use and is continually updated to deploy enhancements and new features.
 
-<br clear="both">
+- [Release notes](https://www.ustaxcourt.gov/release_notes.html) for DAWSON
+- [How we work](./docs/how-we-work.md) has our principles, product team, technical strategy, meetings cadence, tools, etc.
+- [Onboard](https://github.com/ustaxcourt/ef-cms/issues/new?template=onboarding.md) or [offboard](https://github.com/ustaxcourt/ef-cms/issues/new?template=offboarding.md) a teammate
+- [Documentation](./docs/README.md) about the CI/CD setup, API, style guide, UX, code review, etc.
+- [Ongoing development documentation](./wikiwiki/README.md) such as designs, research data, user workflows, etc.
+- [Product roadmap](https://docs.google.com/document/d/1g3D1zPNQqVsWhJ6uGIVcHA04IdBHgYcOCm9roaSfFNk/edit) 🔒
+- [Goals of DAWSON](https://docs.google.com/document/d/1xCh1hbXjJItOWlQ2MH-GDDc0KWho88KKcHfUJcpHkSs/edit) 🔒
 
-The fork of this project in which the bulk of development is occurring is [Flexion’s fork](https://github.com/flexion/ef-cms), but all changes are pulled into the U.S. Tax Court’s repository every two weeks.
+### Backlog
 
-Artifacts for ongoing development such as designs, research data, user workflows etc. are located in the [wiki](https://github.com/flexion/ef-cms/wiki). See also [documentation about how we work](https://github.com/ustaxcourt/ef-cms/blob/staging/docs/how-we-work.md): our principles, product team, technical strategy, meetings cadence, tools, etc.
+| Backlog | Location
+|---------|----------
+| User stories and feature work <br>(Primary backlog) | [GitHub issues in Flexion’s repository](https://github.com/flexion/ef-cms/issues#workspaces/flexionef-cms-5bbe4bed4b5806bc2bec65d3/board?repos=152320868); scrum board visible using the [ZenHub browser plugin](https://www.zenhub.com/extension).
+| Refactoring and bugs | [Trello cards in Flexion’s OpEx/DevEx board](https://trello.com/b/9tgrIFfA/ef-cms-opex-devex). 🔒
```

Comment:
This trello board is also not visible to the general public from what I can see.